dev/.vagrant/machines/default/virtualbox/action_provision

```diff
@@ -0,0 +1 @@
+1.5:2e635e94-ca0e-49d3-b579-385bb4378ed2
```

Comment:
Let's not ship all this `.vagrant` stuff :P You should probably update the .gitignore too.